ci-operator/step-registry/ovn/conf/baremetal-dualstack/ovn-conf-baremetal-dualstack-commands.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -f "${SHARED_DIR}/install-config.yaml" ]; then
+    curl -L https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
if you use `cli-yq` as the image in the .yaml for this step you'll get it for free. it's based off of the ocp cli image
which is based off of rhel8. that should work right?

so you'd want:

```
  from_image:
    name: cli-yq
    namespace: ocp
    tag: latest
```

Reply:
Done! Thanks, that indeed simplifies the logic

Reply:
Is it as simple as `/tmp/yq` instead of `yq` or do I miss something more?

Reply:
yeah, it should be that simple.